src/plugins/kibana_react/public/page_template/page_template_inner.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent } from 'react';
+
+import { EuiEmptyPrompt, EuiPageTemplate } from '@elastic/eui';
+import { withSolutionNav } from './with_solution_nav';
+import { KibanaPageTemplateProps } from './page_template';
+import { getClasses } from './util';
+
+/**
+ * A thin wrapper around EuiPageTemplate with a few Kibana specific additions
+ */
+export const KibanaPageTemplateInner: FunctionComponent<KibanaPageTemplateProps> = ({
```

Comment:
nit: Even though this component is using `KibanaPageTemplateProps`, it probably shouldn't do so directly.

```
type Props = KibanaPageTemplateProps;

export const KibanaPageTemplateInner: FC<Props> ...
```